ports/zephyr/main.c

```diff
@@ -62,6 +66,30 @@
 
 static char heap[MICROPY_HEAP_SIZE];
 
+#if defined(CONFIG_USB_DEVICE_STACK_NEXT)
+
+extern struct usbd_context *mpy_usbd_init_device(usbd_msg_cb_t msg_cb);
+
+static struct usbd_context *mpy_usbd;
+
+static int mp_usbd_init(void) {
```

Comment:
I'm curious why you put this helper function here in main.c, and not in src/usbd.c?  Maybe it can be moved to src/usbd.c?

Reply:
Yea it could, will do.

Reply:
done